api/src/test/java/org/openmrs/api/CohortServiceTest.java

```diff
@@ -65,7 +65,7 @@ public void getCohort_shouldOnlyGetNonVoidedCohortsByName() throws Exception {
 		// now do the actual test: getCohort by name and expect a non voided cohort
 		Cohort exampleCohort = service.getCohort("Example Cohort");
 		assertNotNull(exampleCohort);
-		assertEquals(2, exampleCohort.size());
+		assertEquals(1, exampleCohort.size());
```

Comment:
So how comes the test was passing before? Did you remove any member from the test dataset?